.github/workflows/pytest.yml

```diff
@@ -44,7 +44,7 @@ jobs:
     env:
       PYTEST_SENTRY_DSN: ${{secrets.PYTEST_SENTRY_DSN}}
 
-    timeout-minutes: 10
+    timeout-minutes: 15
```

Comment:
The timeout-minutes are updated on main but it was necessary to update here because the tests were consistently crossing 10 mins timeout on this branch.